README.md

```diff
@@ -201,48 +282,62 @@ EdDSA, as applied to JOSE (including JWT), is described in [RFC 8037](https://da
 {
   "alg": "EdDSA",
   "typ": "JWT",
-  "ucv": "0.8.1"
+  "ucv": "0.9.0"
 }
 ```
 
 ## 3.2 Payload
 
 The payload MUST describe the authorization claims, who is involved, and its validity period.
 
-| Field | Type       | Description                                      | Required |
-| ----- | ---------- | ------------------------------------------------ | -------- |
-| `iss` | `String`   | Issuer DID (sender)                              | Yes      |
-| `aud` | `String`   | Audience DID (receiver)                          | Yes      |
-| `nbf` | `Number`   | Not Before UTC Unix Timestamp (valid from)       | No       |
-| `exp` | `Number`   | Expiration UTC Unix Timestamp (valid until)      | Yes      |
-| `nnc` | `String`   | Nonce                                            | No       |
-| `fct` | `Json[]`   | Facts (asserted, signed data)                    | No       |
-| `prf` | `String[]` | Proof of delegation (witnesses are nested UCANs) | Yes      |
-| `att` | `Json[]`   | Attenuations                                     | Yes      |
+| Field | Type            | Description                                      | Required |
+| ----- | --------------- | ------------------------------------------------ | -------- |
+| `iss` | `String`        | Issuer DID (sender)                              | Yes      |
+| `aud` | `String`        | Audience DID (receiver)                          | Yes      |
+| `nbf` | `Number`        | Not Before UTC Unix Timestamp (valid from)       | No       |
+| `exp` | `Number | null` | Expiration UTC Unix Timestamp (valid until)      | Yes      |
```

Comment:
```suggestion
| `exp` | `Number \| null` | Expiration UTC Unix Timestamp (valid until)      | Yes      |
```
The rendering on this line is wrong, this should make it better

<img width="924" alt="Screen Shot 2022-09-01 at 11 10 02" src="https://user-images.githubusercontent.com/314190/187889629-74ab22a7-d158-46dc-bc0f-427be8b7f265.png">

Reply:
Thanks @hugomrdias!